### PR TITLE
CA-384979 replace XenMotion with storage live migration

### PR DESCRIPTION
**Pull request by lindig**

### CODE REVIEW

**Review by psafont — APPROVED** [review]
It's still mentioned in the features 😬

```
ocaml/xapi-types/features.ml
107:  ; (Xen_motion, ("restrict_xen_motion", Negative, "XenMotion"))

ocaml/xapi-types/features.mli
45:  | Storage_motion  (** Enable Storage XenMotion feature *)
53:  | Xen_motion  (** Enable XenMotion feature *)
```

File: ocaml/xapi-types/features.ml [hunk 1]
```diff
@@ -104,7 +104,7 @@ let keys_of_features =
   ; ( PCI_device_for_auto_update
     , ("restrict_pci_device_for_auto_update", Negative, "PciAU")
     )
-  ; (Xen_motion, ("restrict_xen_motion", Negative, "XenMotion"))
+  ; (Xen_motion, ("restrict_xen_motion", Negative, "Storage_live_migration"))
```
Comment (robhoes) [inline ocaml/xapi-types/features.ml:107]: This is not _storage_ live migration, but plain live migration, without storage.

File: ocaml/xapi-types/features.mli [hunk 2]
```diff
@@ -42,15 +42,15 @@ type feature =
...
+  | Storage_motion  (** Enable Storage Live Migration feature *)
   | VGPU  (** Enable use of virtual GPUs *)
   | Integrated_GPU  (** Enable use of integrated GPU passthrough *)
   | VSS  (** Enable use of VSS *)
   | Guest_agent_auto_update
       (** Enable use of the Windows guest agent auto-update feature. *)
   | PCI_device_for_auto_update
       (** Enable making new VMs with the PCI device that triggers Windows Update. *)
-  | Xen_motion  (** Enable XenMotion feature *)
+  | Xen_motion  (** Enable storage live migration feature *)
```
Comment (robhoes) [inline ocaml/xapi-types/features.mli:53]: as above

File: ocaml/xapi-types/features.ml (hunk 1 above)
Comment (lindig) [reply ocaml/xapi-types/features.ml:107]: Will fix

**Review by psafont — APPROVED** [review]

**Review by snwoods — APPROVED** [review]

**Pull request merged**